app/interactions/games/create_neutral_force_tokens.rb

```diff
@@ -29,11 +35,19 @@ def determine_neutral_force_range(range)
       end
     end
 
-    def create_neutral_force(range_of_houses, territory)
-      range_of_houses.map do |range|
-        next if determine_neutral_force_range(range).blank?
-        game.neutral_force_tokens.create(territory: territory)
-      end
+    def format_player_range(range)
+      players_numbers = Array(range)
+      return players_numbers.first if players_numbers.one?
+      "#{players_numbers.first}-#{players_numbers.last}"
```

Comment:
since we are sanitizing the range of houses before committing to the DB, we could use this format ( `range: "4-6"`) in the YAML, so the sanitizing doesn't have to happen

Reply:
but sometimes we have to check if the number of players is inside the range